src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift

```diff
@@ -376,15 +376,21 @@ enum AppStrings {
 		static let body1 = NSLocalizedString("SurveyConsent_Body1", comment: "")
 		static let body2 = NSLocalizedString("SurveyConsent_Body2", comment: "")
 		static let body3 = NSLocalizedString("SurveyConsent_Body3", comment: "")
-		static let legalTitle = NSLocalizedString("SurveyConsent_Legal_Title", comment: "")
-		static let legalBody1 = NSLocalizedString("SurveyConsent_Legal_Body1", comment: "")
-		static let legalBody2 = NSLocalizedString("SurveyConsent_Legal_Body2", comment: "")
+		static let legalTitle = NSLocalizedString("SurveyConsent_LegalTitle_BoldText", comment: "")
+		static let legalBody1 = NSLocalizedString("SurveyConsent_LegalBody1", comment: "")
+		static let legalBody2 = NSLocalizedString("SurveyConsent_LegalBody2", comment: "")
```

Comment:
If these texts are legal texts then these texts (DE/EN/TU) have to be placed file in Localizable.legal.strings.

Reply:
I moved the legal texts to the legal localizable files and added Turkish and English translation as well.